timescaledb/quick-start/ruby.md

```diff
@@ -4,727 +4,707 @@ excerpt: Get started with TimescaleDB for a Ruby application
+    path: nil,
+    performance: nil>
+    ```
+
+    This example uses only the **real** performance from [benchmark] but you can
+    collect additional metrics to see more details about your system.
+
+</Collapsible>
+
+</Procedure>
+
+## Explore aggregation functions
+
+Now that you know what pages exist, you can explore the results. You can go 
+page by page, or all pages together, and group by path or not:
+
+<Procedure>
+
+<Collapsible heading="Exploring aggregation functions" headingLevel={3} defaultExpanded={false}>
+
+1.  In the `page_load.rb` file located at `my_app/app/models` directory, add
+    these scopes, for average response time, `min` and `max` requests, and
+    collect unique paths from page loads:
+
+    ```ruby
+    class PageLoad < ApplicationRecord
+      scope :per_minute, -> { time_bucket('1 minute') }
+      scope :per_hour, -> { time_bucket('1 hour') }
+      scope :per_day, -> { time_bucket('1 day') }
+      scope :per_week, -> { time_bucket('1 week') }
+      scope :per_month, -> { time_bucket('1 month') }
+      scope :average_response_time_per_minute, -> { time_bucket('1 minute', value: 'avg(performance)') }
+      scope :average_response_time_per_hour, -> { time_bucket('1 hour', value: 'avg(performance)') }
+      scope :worst_response_time_last_minute, -> { time_bucket('1 minute', value: 'max(performance)') }
+      scope :worst_response_time_last_hour, -> { time_bucket('1 hour', value: 'max(performance)') }
+      scope :best_response_time_last_hour, -> { time_bucket('1 hour', value: 'min(performance)') }
+      scope :paths, -> { distinct.pluck(:path) }
+      scope :time_bucket, -> (time_dimension, value: 'count(1)') {
+        select("time_bucket('#{time_dimension}', created_at), #{value})")
+      end
+     ```
+
+1.  In the Ruby console,to collect unique paths from page loads:
+
+    ```ruby
+     PageLoad.paths # => ["/page_loads/new", "/static_pages/home"]
+    ```
+
+    The result is similar to:
+
+    ```ruby
+    PageLoad Pluck (276.1ms)  SELECT DISTINCT "page_loads"."path" FROM "page_loads"
+    => [nil, "/static_pages/home"]
+    ```
+
+1.  In the Ruby console, to get the actual metrics generated for the response
+    time filtering by methods that contains `response_time` use:
+
+    ```ruby
+    PageLoad.methods.grep /response_time/
+    ```
+
+    The result is similar to:
+
+    ```ruby
+    PageLoad.methods.grep /response_time/
+    # => [:average_response_time_per_hour,
+    #  :average_response_time_per_minute,
+    #   :worst_response_time_last_hour,
+    #   :worst_response_time_last_minute,
+    #   :best_response_time_last_hour]
+     ```
+
+</Collapsible>
+
+</Procedure>
+
+## Next steps
+
+Now that you're able to connect, read, and write to a TimescaleDB instance from
+your Ruby application, and generate the scaffolding necessary to build a new
+application from an existing TimescaleDB instance, be sure to check out these
```

Comment:
If the person is starting with Ruby, maybe they'll not be familiar with other languages. I'd just make sure that they get some more ruby-oriented content. Here are a few suggestions:

https://evilmartians.com/chronicles/time-series-data-using-timescaledb-with-ruby-on-rails

I also wrote a few tutorials for who that are interested in adopting a gem with a seamless integration:


https://ideia.me/using-the-timescale-gem-with-ruby
https://ideia.me/timescale-continuous-aggregates-with-ruby
https://ideia.me/hierarchical-continuous-aggregates-with-ruby

And the documentation of my gem also contains several full feature examples:

https://jonatas.github.io/timescaledb/#examples


This one is also the source of some of my talks:
https://jonatas.github.io/timescaledb/toolkit_lttb_tutorial/